web/src/main/java/se/inera/intyg/webcert/web/service/facade/util/UtkastToCertificateConverterImpl.java

```diff
@@ -136,6 +136,10 @@ private Certificate convertToCertificate(Utkast certificate) {
                 certificateToReturn.getMetadata().getTypeVersion())
         );
 
+        certificateToReturn.getMetadata().setAvailableForCitizen(
+            !(certificate.getIntygsTyp().equals("db") || certificate.getIntygsTyp().equals("doi"))
```

Comment:
You can use the constants in DoiModuleEntryPoint and DbModuleEntryPoint instead of hardcoding it.

Reply:
ok